src/AssetsPage/AssetsPage.scss

```diff
@@ -0,0 +1,3 @@
+.assets {
+	
+}
```

Comment:
I left this in place because I'm assuming we will need some overarching page styles and that this will be used soon anyhow (it used to house the pink).